internal/test/scorecard/common_utils.go

```diff
@@ -536,6 +515,33 @@ func (r *TestResources) updateAndWaitTillCryostatAvailable(cr *operatorv1beta2.C
 	return cr, err
 }
 
+func (r *TestResources) updateStorageOptions(ctx context.Context, cr *operatorv1beta2.Cryostat) (*operatorv1beta2.Cryostat, error) {
+	result := &operatorv1beta2.Cryostat{}
```

Comment:
I guess we don't need to initialize an empty CR? Perhaps, `var result *operatorv1beta2.Cryostat`?

This way, if any error, `nil` is returned and access to nil can throw nil pointer exception. This way, it is easier to understand other than an empty CR like this issue?

Reply:
Good call